

lib/kobject_uevent.c:39: warning: `action_to_string' defined but not used

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/lib/kobject_uevent.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN lib/kobject_uevent.c~kobject_uevent-warning-fix lib/kobject_uevent.c
--- 25/lib/kobject_uevent.c~kobject_uevent-warning-fix	2004-10-05 02:11:18.368795080 -0700
+++ 25-akpm/lib/kobject_uevent.c	2004-10-05 02:11:18.372794472 -0700
@@ -23,6 +23,7 @@
 #include <linux/kobject.h>
 #include <net/sock.h>
 
+#if defined(CONFIG_KOBJECT_UEVENT) || defined(CONFIG_HOTPLUG)
 /* 
  * These must match up with the values for enum kobject_action
  * as found in include/linux/kobject_uevent.h
@@ -42,6 +43,7 @@ static char *action_to_string(enum kobje
 	else
 		return actions[action];
 }
+#endif
 
 #ifdef CONFIG_KOBJECT_UEVENT
 static struct sock *uevent_sock;
_
